game/templates/game/opening_detail.html

```
@@ -0,0 +1,67 @@
+    </p>
+
+    <div id="trainer-feedback" class="feedback-box">
+        Ready to begin.
+    </div>
+
+    <div class="progress-box">
+        <div id="move-progress">
+            0 / {{ opening.moves|length }}
+        </div>
+        <input
+            type="text"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)